projects/plugins/jetpack/extensions/blocks/sharing-button/variations.js

```diff
@@ -29,8 +29,8 @@ export const variations = [
 	},
 	{
 		name: 'mail',
-		attributes: { service: 'mail', label: 'Mail' },
-		title: 'Mail',
+		attributes: { service: 'mail', label: __( 'Mail', 'jetpack' ) },
+		title: __( 'Mail', 'jetpack' ),
```

Comment:
I'm thinking it may be useful to add context to those strings to help translators understand how this is meant to be used (e.g. "verb"?)